client/jetpack-cloud/sections/partner-portal/stored-credit-card-v2/index.tsx

```diff
@@ -50,13 +50,15 @@ export default function StoredCreditCardV2( {
 					paymentMethodId: creditCard.id,
 					useAsPrimaryPaymentMethod: true,
 				} );
+				recordTracksEvent( 'calypso_partner_portal_payments_card_actions_set_as_primary_click' );
 			},
 		},
 		{
 			name: translate( 'Delete' ),
 			isEnabled: true,
 			onClick: () => {
 				setIsDeleteDialogVisible( true );
+				recordTracksEvent( 'calypso_partner_portal_payments_card_actions_delete_click' );
```

Comment:
Same as above. We will have to dispatch these event-tracking calls.

Reply:
I've updated both and thanks!